main.go

```diff
@@ -98,16 +98,17 @@ func main() {
 	ctrl.SetLogger(utils.GetDefaultLogger(logLevel))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                        scheme,
-		MetricsBindAddress:            metricsAddr,
-		Port:                          9443,
-		HealthProbeBindAddress:        probeAddr,
+		Scheme:                 scheme,
+		Metrics:                metricsserver.Options{BindAddress: metricsAddr},
+		HealthProbeBindAddress: probeAddr,
+		WebhookServer: webhook.NewServer(webhook.Options{
+			Port: 9443,
+		}),
 		LeaderElection:                enableLeaderElection,
 		LeaderElectionID:              "hetzner.cluster.x-k8s.io",
 		LeaderElectionNamespace:       leaderElectionNamespace,
 		LeaderElectionResourceLock:    "leases",
 		LeaderElectionReleaseOnCancel: true,
-		Namespace:                     watchNamespace,
```

Comment:
why do you remove this? CAPI still uses it https://github.com/kubernetes-sigs/cluster-api/blob/main/main.go#L301

Reply:
I get:

> unknown field Namespace in struct literal of type manager.Options

From controller-runtime release page:

> cache.Options.Namespace has been removed in favor of cache.Options.Namespaces

Reply:
yes just check out the CAPI code!

Reply:
According to the [contract](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/support-multiple-instances#contract), it is required. I added it again.